src/OpenSage.Game/LanguageSetting.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OpenSage.Data;
+using OpenSage.Utilities;
+
+namespace OpenSage
+{
+    public static class LanguageSetting
+    {
+        //do we need a default?
+        private const string DefaultLanguage = "englisch";
+        private static string _detectedLanguage;
```

Comment:
I don't think we should use a static field for this...

Looking up the language setting in e.g `ContentManager`'s constructor would probably make more sense. The language is only used twice, in `TranslationManager` and `WindowLoader`, so we don't need to store it for later. You can move `ReadFromRegistry` there and rename it.

Reply:
Also the default language is "english" not "englisch". You've probably accidentally used the german version of "english"